src/libraries/System.Text.RegularExpressions/tests/RegexParserTests.cs

```diff
@@ -709,23 +709,34 @@ public void ParseCheckOffset(string pattern, RegexOptions options, RegexParseErr
             Parse(pattern, options, error, offset);
         }
 
+
         private static void Parse(string pattern, RegexOptions options, RegexParseError? error, int offset = -1)
         {
             if (error != null)
             {
                 Assert.InRange(offset, 0, int.MaxValue);
-                Throws(error.Value, offset, () => new Regex(pattern, options));
+                Throws(pattern, options, error.Value, offset, () => new Regex(pattern, options));
                 return;
             }
 
             Assert.Equal(-1, offset);
+            LogActual(pattern, options, RegexParseError.Unknown, -1);
 
             // Nothing to assert here without having access to internals.
             new Regex(pattern, options); // Does not throw
 
             ParsePatternFragments(pattern, options);
         }
 
+        private static void LogActual(string pattern, RegexOptions options, RegexParseError error, int offset)
+        {         
+            string s = (error == RegexParseError.Unknown) ?
+                @$"        [InlineData(@""{pattern}"", RegexOptions.{options.ToString()}, null)]" :
+                @$"        [InlineData(@""{pattern}"", RegexOptions.{options.ToString()}, RegexParseError.{error.ToString()}, {offset})]";
+
+            // File.AppendAllText(@"/tmp/out.cs", s + "\n"); // for updating baseline
+        }
```

Comment:
I don't understand... what's the purpose of this method?

Reply:
It updates the baseline: I can take a bunch of presumably-invalid regex inputs from someplace, and turn them into this format:
```
[InlineData("SOMEREGEX1", RegexOptions.None, null)]
[InlineData("SOMEREGEX2", RegexOptions.None, null)]
...
```

I paste those into the test, uncomment this line, and run the test, and it emits into that file exactly what I need to paste in  eg
```
[InlineData(@"SOMEREGEX1", RegexOptions.None, RegexParseError.UnrecognizedEscape, 3)]
[InlineData(@"SOMEREGEX2", RegexOptions.None, InsufficientClosingParentheses, 2)]
...
```
I then replace what I pasted in with those, and I have my updated test.

Reply:
is that OK? I'm finding this a useful approach: have the test construct the baseline. 

Of course, I'll eyeball to satisfy myself that the results are good. In some cases, I'm emitting in a different format locally to diff against the original test output as well.

Reply:
Added a comment.

Reply:
Is it possible to do easily as a standalone, commented out test?  I think what I'm objecting to is it's modifying other tests with dead code that is almost certain to rot.

Reply:
Sure, I can change it to that.